h/static/styles/thread.scss

```diff
@@ -48,15 +61,22 @@ $thread-padding: $annotation-card-left-padding;
 
   & > ul {
     padding-left: $thread-padding + 3px;
+    padding-right: $thread-padding + 3px;
     margin-left: -$thread-padding;
+    margin-right: -$thread-padding;
   }
 
   // nested threads for annotation replies
   .thread {
-    border-left: 1px dotted $gray-light;
+    border-left: 1px dotted $grey-3;
     padding: 0;
     padding-left: $thread-padding;
-    margin-top: 10px;
+
+    &:hover {
+      & > .threadexp > span {
```

Comment:
I'm going to merge this branch now but I suggest that we collaborate on getting the reply hover states implemented afterwards and clear this up afterwards.

As a general note, it is usually always preferable to try to simplify the CSS selectors down to just one or two classes if you can. If not, a comment explaining the desired visual effect is very helpful for future reference.